src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.Android.cs

```diff
@@ -581,6 +581,9 @@ private int ComputeScrollRange(RecyclerView.State state)
 			var remainingLines = remainingItems / leadingLine.NumberOfViews;
 			var remainingItemExtent = remainingLines * leadingLine.Extent;
 
+			int headerExtent = HeaderViewCount > 0 ? GetChildEndWithMargin(GetChildAt(GetHeaderViewIndex())) - GetChildStartWithMargin(GetChildAt(GetHeaderViewIndex())) : 0;
```

Comment:
comment from earlier PR applies, use GetChildExtentWithMargin() instead of End-Start

Reply:
done